test/e2e/autoscale_test.go

```diff
@@ -370,10 +371,8 @@ func TestTargetBurstCapacity(t *testing.T) {
 	cancel := logstream.Start(t)
 	defer cancel()
 
-	ctx := setup(t, autoscaling.KPA, autoscaling.Concurrency,
+	ctx := setup(t, autoscaling.KPA, autoscaling.Concurrency, 10, targetUtilization,
```

Comment:
here and below, we're losing the key of "10", so comment what it is `10 /* target concurrency*/`.

Reply:
Done